db/migrate/20220401131950_create_posts.rb

```diff
@@ -1,6 +1,10 @@
 class CreatePosts < ActiveRecord::Migration[7.0]
   def change
     create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :commets_counter, default: 0
+      t.integer :likes_counter, default: 0
 
```

Comment:
Please add migration for `author_id` as foreign key in `posts` table as shown in [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).